#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_CMakeVersion.dpatch by <ubuntu@cafuego.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Updates src/CMakeLists.txt

@DPATCH@

--- old/bzrversion.sh 1970-01-01 10:00:00.000000000 +1000
+++ new/bzrversion.sh 2014-01-21 10:31:04.457979439 +1100
@@ -0,0 +1,2 @@
+#!/bin/sh
+dpkg-parsechangelog -l./debian/changelog | egrep '^Version:' | cut -f2 -d '+'
--- old/src/CMakeLists.txt  2014-01-21 10:24:53.846135434 +1100
+++ new/src/CMakeLists.txt  2014-01-21 10:33:59.748538164 +1100
@@ -14,6 +14,11 @@
 project(inkscape)

 set(INKSCAPE_VERSION 0.48+devel)
+
+set(VER_PREFIX "${INKSCAPE_VERSION}")
+execute_process(COMMAND ./bzrversion.sh WORKING_DIRECTORY .. OUTPUT_VARIABLE VER_BZRREV ERROR_QUIET)
+set(INKSCAPE_VERSION "${VER_PREFIX}r${VER_BZRREV}")
+
 set(PROJECT_NAME inkscape)
 set(CMAKE_INCLUDE_CURRENT_DIR TRUE)
